### PR TITLE
Tests: add component tests for tx history filter

### LINKED ISSUE #3759: Tx history filter component automation tests
In order to increase our automation tests coverage, component tests for Tx history filter were added.

### PR DESCRIPTION
**Pull request by mike10ca**
## What it solves

Resolves #3759 

## How this PR fixes it

- Add component tests for TxFilterForm

## How to test it

- Run tests and observe outcome

### CODE REVIEW

File: src/components/transactions/TxFilterForm/TxFilterForm.test.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    expect(await screen.findByText(errorMsgZero, { selector: 'label' })).toBeInTheDocument()
+  })
+
+  it('Verify that entering negative numbers and a non-numeric value in the amount filter is not allowed', async () => {
+    renderComponent()
+
+    const amountInput =
+      screen.getByTestId('amount-input').querySelector('input') ??
+      (() => {
+        throw new Error('Input element not found')
```
Comment (compojoom) [inline src/components/transactions/TxFilterForm/TxFilterForm.test.tsx:133]: why are you throwing here? On the next line you expect this element to be in the document?

Comment (mike10ca) [reply src/components/transactions/TxFilterForm/TxFilterForm.test.tsx:133]: I had the error `Type 'null' is not assignable to type 'Element | Node | Document | Window'.ts(2345)` which was solved by adding that statement.

Comment (compojoom) [reply src/components/transactions/TxFilterForm/TxFilterForm.test.tsx:133]: that is strange

Comment (mike10ca) [reply src/components/transactions/TxFilterForm/TxFilterForm.test.tsx]: changed, thank you for feedback

**Review by compojoom — APPROVED** [review]
👍

**Pull request merged**